sdscli/adapters/hysds/fabfile.py

```diff
@@ -917,11 +925,29 @@ def send_hysds_ui_conf():
     upload_template('index.template.js', dest_file, use_jinja=True, context=get_context('mozart'),
                     template_dir=os.path.join(ops_dir, 'mozart/ops/hysds_ui/src/config'))
 
+    user_path = get_user_files_path()
 
-def send_hysds_ui_conf():
-    dest_file = '~/mozart/ops/hysds_ui/src/config/index.js'
-    upload_template('index.template.js', dest_file, use_jinja=True, context=get_context('mozart'),
-                    template_dir=os.path.join(ops_dir, 'mozart/ops/hysds_ui/src/config'))
+    tosca_cfg = '~/mozart/etc/tosca.js'
+    if os.path.exists(os.path.join(user_path, 'tosca.js')):
+        send_template_user_override('tosca.js', tosca_cfg, node_type='mozart')
```

Comment:
should we also add a print statement to indicate whether we're pulling in a custom or default tosca?